lib/capistrano/recipes/deploy/strategy/jenkins_artifact.rb

```diff
@@ -4,23 +4,34 @@
 require 'jenkins_api_client'
 
 class ::JenkinsApi::Client::Job
+  def get_last_successful_build(job_name)
+    @logger.info "Obtaining last successful build number of #{job_name}"
+    @client.api_get_request("/job/#{path_encode(job_name)}/lastSuccessfulBuild")
+  end
+
   def get_last_successful_build_number(job_name)
     @logger.info "Obtaining last successful build number of #{job_name}"
     res = @client.api_get_request("/job/#{path_encode(job_name)}/lastSuccessfulBuild")
     res['number']
   end
 
   def find_artifact_with_path(job_name, relative_path)
```

Comment:
[nitp]
To match with `find_last_successful_artifact`, name of this method can be similar one?